hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmDirectoryInfo.java

```diff
@@ -235,6 +229,17 @@ public static OmDirectoryInfo getFromProtobuf(DirectoryInfo dirInfo) {
     if (dirInfo.hasOwnerName()) {
       opib.setOwner(dirInfo.getOwnerName());
     }
+    return opib;
+  }
+
+  /**
+   * Parses DirectoryInfo protobuf and creates OmPrefixInfo.
+   * @param dirInfo
+   * @return instance of OmDirectoryInfo
+   */
```

Comment:
This javadoce does not provide any useful information and generate "tag description is missing" warning.  Please remove it or rewrite it.